packages/math/base-elements.lua

```diff
@@ -1367,15 +1392,40 @@ function elements.sqrt:shape ()
    end
    self.extraAscender = constants.radicalExtraAscender * scaleDown
 
-   -- HACK: More or less ad hoc values, see output method for more details
-   self.symbolWidth = SILE.shaper:measureChar("√").width
-   self.symbolHeight = SILE.types.length("1.1ex"):tonumber() * scaleDown
-
-   self.width = self.radicand.width + SILE.types.length(self.symbolWidth)
-   self.height = self.radicand.height + self.radicalVerticalGap + self.extraAscender
+   -- HACK: We draw own own radical sign in the output() method.
+   -- Derive dimensions for the radical sign (more or less ad hoc).
+   -- Note: In TeX, the radical sign extends a lot below the baseline,
+   -- and MathML Core also has a lot of layout text about it.
+   -- Not only it doesn't look good, but it's not very clear vs. OpenType.
+   local radicalGlyph = SILE.shaper:measureChar("√")
+   local ratio = (self.radicand.height:tonumber() + self.radicand.depth:tonumber()) / (radicalGlyph.height + radicalGlyph.depth)
+   self.symbolHeight = SILE.types.length(radicalGlyph.height)
+   self.symbolDepth = SILE.types.length(radicalGlyph.depth) + (ratio > 1 and math.log(ratio) or 0) * self.radicalVerticalGap * scaleDown
+   self.symbolWidth = SILE.types.length(radicalGlyph.width) + (ratio > 1 and math.log(ratio) or 0) * self.radicalVerticalGap * scaleDown
```

Comment:
Missing scaleDown on the height, and needing parentheses on the depth and width

Noticed during tests for #2151 - This notably slightly affects inline display (text mode).

Now:

![image](https://github.com/user-attachments/assets/b8b22f1d-84c0-4dbf-9205-419afe8d82cb)

With scaleDown fixed:

![image](https://github.com/user-attachments/assets/b63959c8-4980-434d-a776-1521a691fb95)

Reply:
Done, commit amended